.build/run-tests.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
Was this an accidental change or intentional? If intentional (for my own education) - why the change?

Reply:
I saw it break in some environments (where it picked up custom bash command on the path).
this was the simple fix. i haven't yet heard of an env where /bin/bash does not exist.